Readme.md

```diff
@@ -1,7 +1,30 @@
-# Nuklear
-
-[![](https://github.com/Immediate-Mode-UI/Nuklear/workflows/C%2FC++%20CI/badge.svg )](https://github.com/Immediate-Mode-UI/Nuklear/actions)
-
+<h1 align="center">Nuklear</h1>
+<h3 align="center">A small, simple and portable immediate mode UI library</h3>
+
+<p align="center">
+  <a href="https://github.com/Immediate-Mode-UI/Nuklear/wiki">Documentation</a> |
+  <a href="https://github.com/Immediate-Mode-UI/Nuklear/wiki">Getting Started</a> |
+  <a href="#Gallery">Gallery & Examples</a>
+</p>
+
+<p align="center">
+  <a href="https://github.com/Immediate-Mode-UI/Nuklear/actions">
+    <img src="https://github.com/Immediate-Mode-UI/Nuklear/workflows/C%2FC++%20CI/badge.svg" alt="buildstatus"/>
+  </a>
+  <a href="https://discord.gg/r27Ze67R">
+    <img alt="discord-badge" src="https://img.shields.io/discord/922234432071548968"/>
+  </a>
+<br>
+  <a href="https://discord.gg/r27Ze67R">
+    Join us on Discord!
+  </a>
+</p>
+
+<p align="center">
+  <img src="https://cloud.githubusercontent.com/assets/8057201/11761525/ae06f0ca-a0c6-11e5-819d-5610b25f6ef4.gif" alt="drawing" width="800"/>
+</p>
```

Comment:
Not sure how I feel about embedded all this HTML in the Markdown file...

1. It isn't [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp)
2. Many of the img source URLs could easily change
3. Makes reading the markdown itself somewhat challenging

Reply:
I generally agree wi should not rely on external sources and shall put e.g. images directly in this repo.

Regarding HTML in markdown I'm not that much afraid of such shorter snippets if they improve the documentation understanding. We can always improve on this later.

Still, making the HTML semantic would be nice (though IMHO by far not a requirement).

Reply:
- I agree it might look dodgy at first, but default vanilla markdown unfortunately does not allow us to do certain formatting, thats why i tend to keep it small, constraint to "header" like content when used in markdown.
- I also fully agree we should keep image sources in the repository. The images used are from this repository aside from the discord badge. Im not  sure if we can bring that in here aswell. Shields.io creates this nice "live" github like badge for many different services, but we can also drop it and just use a link.
- Legibilty is something I can try to tweak on, iIll be back later this week and will try to make some adjustments. Semantic elements should work fine here aswell 👍.